src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,49 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DEFAULT_VALID_AGE = 18;
+    private static final int DEFAULT_VALID_PASSWORD = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        userValidator(user);
+        loginValidator(user);
+        ageValidator(user);
+        passwordValidator(user);
+        return storageDao.add(user);
+    }
+
+    private void userValidator(User user) {
```

Comment:
UserValidator is a good name for class. For methods we use verbs. Method is ACTION.
`validateUser()` would be better.